terraform/environments/analytical-platform-ingestion/kms-keys.tf

```
@@ -240,3 +240,16 @@ module "s3_datasync_kms" {
 
   deletion_window_in_days = 7
 }
+
+module "common_secretsmanager_secret" {
```

```suggestion
module "secretsmanager_common_kms" {
```
This will need updating in references